content/updates/2017-08-08-max-gov-migration-instructions.md

```diff
@@ -0,0 +1,91 @@
++++
+date = "2017-08-08"
+title = "Announcement: cloud.gov accounts will migrate to OMB MAX"
++++
+
+cloud.gov is migrating all users with native cloud.gov accounts (not users with an agency single sign on provider) to use OMB MAX for authentication to the platform. This page is a guide to what will change and how you can prepare for this transition.
+
+## What’s changing?
+
+Right now, there are two ways to log in to cloud.gov: either you use your agency’s credentials or a cloud.gov account. This change only affects users who have a cloud.gov account.
+
+<img src="/img/login-old.png" alt="the current cloud.gov login screen" width="624" height="622" />
+
+In September 2017, we will switch users with a cloud.gov account to use an OMB MAX account to log in to cloud.gov. We’re making this change both to increase the compliance of cloud.gov with federal standards and law and to make managing your account easier for you and your agency.
```

Comment:
Can we add a bit more framing around the date here? I think the top priority question/need for people reading this will be "when exactly is this happening".